packages/patternfly-3/patternfly-react/src/components/ExpandCollapse/ExpandCollapse.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName, STORYBOOK_CATEGORY, DOCUMENTATION_URL } from 'storybook/constants/siteConstants';
+import { name } from '../../../package.json';
+import { select, text, withKnobs } from '@storybook/addon-knobs';
+
+import { ExpandCollapse } from './index';
+import { ALIGN_TYPES } from './constants';
+
+const stories = storiesOf(`${storybookPackageName(name)}/${STORYBOOK_CATEGORY.WIDGETS}/Expand Collapse`, module);
+stories.addDecorator(withKnobs);
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Expand Collapse',
+    documentationLink: `${DOCUMENTATION_URL.PATTERNFLY_ORG_FORMS}expand-collapse-section/`
+  })
+);
+
+stories.add(
+  'ExpandCollapse',
```

Comment:
Would show the left/center better if we added a div around this with a set width and a border: 
```
<div style={{ width: '400px', border: '1px solid lightgray' }}
  <ExpandCollapse  ...
```

Reply:
done